django/template/engine.py

```diff
@@ -52,6 +52,18 @@ def __init__(self, dirs=None, app_dirs=False, context_processors=None,
         self.builtins = self.default_builtins + builtins
         self.template_builtins = self.get_template_builtins(self.builtins)
 
+    def __repr__(self):
+        return (
+            '<%(class)s dirs=%(dirs)s, app_dirs=%(app_dirs)s, loaders=%(loaders)s,'
+            ' context_processors=%(context_processors)s>'
+        ) % {
```

Comment:
Please add all initialization parameters to the `__repr__()` and keep the order from `__init__()`. We should also use `repr()` for most attributes, e.g.
```python
    def __repr__(self):
        return (
            f'<{self.__class__.__qualname__} dirs={self.dirs!r} '
            f'app_dirs={self.app_dirs} '
            f'context_processors={self.context_processors!r} '
            f'debug={self.debug} loaders={self.loaders!r} '
            ...
        )
```